tests/e2e/util/commonUtils.go

```diff
@@ -53,10 +53,11 @@ func Shell(format string, args ...interface{}) (string, error) {
 	glog.V(2).Infof("Running command %s", command)
 	c := exec.Command(parts[0], parts[1:]...) // #nosec
 	bytes, err := c.CombinedOutput()
-	glog.V(3).Infof("Command output: \n%s", string(bytes[:]))
+	glog.V(3).Infof("Command output: \n %s, err: %v", string(bytes[:]), err)
```

Comment:
this is not the return code, it is just logs.  No one should be parsing off that.